packages/turbo-utils/__fixtures__/common/old-workspace-config/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "apps/*",
+      "packages/*"
+    ]
```

Comment:
Doesn't look like you _need_ 3 workspaces for this test and this use case. I'd try to keep fixtures as small as possible so it's easier to understand what part of them matters